### PR TITLE
Address tfjs-node typos in documentation strings

### PR DESCRIPTION
**Pull request by gaikwadrahul8**
Hi, Team
I've identified and corrected several typos within the documentation strings (code comments) of `.ts `files in the `tfjs-node` folder. I believe these corrections will improve the clarity and accuracy of the documentation for users. I appreciate your time and look forward to your feedback. Thank you.

### CODE REVIEW

File: tfjs-node/src/run_tests.ts [hunk 1]
```diff
@@ -122,15 +122,15 @@ const IGNORE_LIST: string[] = [
...
+  'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit default dimRoundingMode',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=floor',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=round',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=ceil',
   // tslint:disable-next-line:max-line-length
-  'pool test-tensorflow {} avg x=[3,3,1] f=[3,3] s=3 d=1 p=explicit defualt dimRoundingMode',
+  'pool test-tensorflow {} avg x=[3,3,1] f=[3,3] s=3 d=1 p=explicit default dimRoundingMode',
```
Comment (mattsoulanille) [inline tfjs-node/src/run_tests.ts:133]: These test names may need to be updated in tfjs-core as well, or they might not be excluded in the tfjs-node tests.

**Review by mattsoulanille — APPROVED** [review]
LGTM, but see my comment about tests.

**Comment by gbaned** [comment]
Hi @gaikwadrahul8 Can you please check above @mattsoulanille's [comment](https://github.com/tensorflow/tfjs/pull/8193#discussion_r1511637248)?  Thank you!

File: tfjs-node/src/run_tests.ts (hunk 1 above)
Comment (gaikwadrahul8) [reply tfjs-node/src/run_tests.ts:133]: Hi, @mattsoulanille, @gbaned 

I apologize for the delayed response and I have updated test names in tfjs-core, please refer this PR [#8184](https://github.com/tensorflow/tfjs/pull/8184) and I appreciate your time and look forward to your feedback. Thank you.

**Review by pyu10055 — APPROVED** [review]

**Comment by gaikwadrahul8** [comment]
Hi @mattsoulanille,

I apologize for the delayed response. I have now created three separate PRs https://github.com/tensorflow/tfjs/pull/8411, https://github.com/tensorflow/tfjs/pull/8412, https://github.com/tensorflow/tfjs/pull/8413 to fix typos in the documentation strings of the **tfjs-layers, tfjs-node, and tfjs-core directories**, in order to keep the typo fixes separate for test names for various backends due to which build was failing Therefore, I will now close this PR

Thank you for your cooperation and help.

**Pull request closed without merging**